exHTML/62018_Martin_Veselinov.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8" />
+		<title>Информация</title>
+		<h1>Мартин Веселинов, ФН - 62018</h1>
+	</head>
+	<body>
+		<p>Студент в Софтуерно Инженерство, 3 курс.</p>
+		<h3>Технологии:</h3>
+		<ol>
+			<li>Си Шарп</li>
+			<li>Пайтън</li>
+			<li>Си++</li>
+			<li>Линукс</li>
+			<li>Си</li>
+		</ol>
+		<a href="https://www.facebook.com/martin.veselinov.5" target="blank"> Линк към моя Фейсбук профил. <a/>
```

Comment:
```suggestion
		<a href="https://www.facebook.com/martin.veselinov.5" target="_blank"> Линк към моя Фейсбук профил. <a/>
```
Please change the content of the target attribute. Currently it is not valid.. Reference: https://www.geeksforgeeks.org/html-link-target-attribute/